apps/api/src/modules/notifications/notifications.service.ts

```diff
@@ -34,16 +34,32 @@ export class NotificationsService extends CoreService<Notification> {
     authHeader: Request,
   ): Promise<Notification> {
     this.logger.log('Creating notification...');
+
+    // TODO: Write better code logic, update notification entity
+    // Get channel type from providerId & Set the channelType based on providerEntry
+    const providerEntry = await this.providersService.getById(notificationData.providerId);
```

Comment:
We should also add an additional check here in case providerEntry itself does not exist based on providerId value.

Reply:
Added a check for the same